### DISCUSSION TITLE
New light theme is too contrasty

### DISCUSSION THREAD

**Discussion by DavidBailes** [audacity/audacity#6845] [Development]
This is obviously subjective, but I find the new light theme is too contrasty. There is a lot of very dark for a "light" theme.
I find the old light theme, now the classic theme, much easier on the eyes.

**Comment by petersampsonaudacity** [comment]
For me it is only the background, the empty track-space, that is too dark for a light theme,
and I don't particularly like the black background for the timers:
![image](https://github.com/audacity/audacity/assets/50205984/5057bca0-c5f0-4946-8b49-bcd7a197398b)

Otherwise I'm enjoying the new Light theme - and I've been working with it a LOT for testing and documentation purposes.

**Reply by petersampsonaudacity** [reply]
But the focus indicator in the new Light theme is not contrasted enough:

See: #6702

**Comment by Mr-Personality** [comment]
Independently, I formed the same opinion about the background color. See my recent issue: #7208
